src/main/java/mate/academy/service/impl/MovieServiceImpl.java

```diff
@@ -0,0 +1,25 @@
+package mate.academy.service.impl;
+
+import mate.academy.dao.MovieDao;
+import mate.academy.exception.DataProcessingException;
+import mate.academy.lib.Inject;
+import mate.academy.lib.Service;
+import mate.academy.model.Movie;
+import mate.academy.service.MovieService;
+
+@Service
+public class MovieServiceImpl implements MovieService {
+    @Inject
+    private MovieDao movieDao;
+
+    @Override
+    public Movie add(Movie movie) {
+        return movieDao.add(movie);
+    }
+
+    @Override
+    public Movie get(Long id) {
+        return movieDao.get(id).orElseThrow(() ->
```

Comment:
```suggestion
        return movieDao.get(id).get();
```
this option is also good